### ISSUE TITLE
Allow tox environments to run in windows (i.e. don't rely on bash)

### ISSUE THREAD

**Issue by dhermes** [GoogleCloudPlatform/gcloud-python#1123]
[Currently][1] we **heavily** rely on bash in our `tox.ini`:

- [x] `cover.sh` (#1237)
- [x] `custom_pip_install.sh` (#1350)
- [x] `merge.sh` (#1364 ~~Only run from `.travis.yml`, no longer needed after #1350 is in since `merge.sh` only delegates to `update_docs.sh` after the fact~~)
- [x] `pep8_on_repo.sh` (#1348)
- [x] `run_system_tests.sh` (#1349)
- [ ] `update_docs.sh` (Only run from `merge.sh`)
- [x] `update_wheels_project.sh` (#1350)
- [ ] `Makefile`

----

We could do this by re-thinking those commands and just using shell-agnostic commands in `tox.ini` or we could get someone who knows Windows well to write a PowerShell equivalent for us.

----

A nice [`tox` tricks and patterns][2] post suggests using a `foo.cmd` file and then

```bash
#!/bin/bash -eE

:<<"::batch"
@echo off
powershell -ExecutionPolicy ByPass -File foo.ps1 %*
goto :end
::batch

foo.sh $*
exit $?

:<<"::done"
:end
::done
```

where `foo.ps1` is PowerShell and `foo.sh` is bash.

This way the `::batch` heredoc gets run on Windows and skips over the `foo.sh` part via `goto :end`, while on *nix systems the `::done` heredoc is skipped due to the `exit $?`

**UPDATE**: I wasn't sure what the `:` did before the heredocs but I [see][3] that it acts as a way to comment them out in bash. [This][4] helps explain the Windows side of things; the lines starting with `:` and `::` are ignored, but those with `:` can be jumped to while the others can't.

[1]: https://github.com/GoogleCloudPlatform/gcloud-python/blob/a01961428f34133174a53b52dbe286952841e719/tox.ini
[2]: http://blog.ionelmc.ro/2015/04/14/tox-tricks-and-patterns/
[3]: http://stackoverflow.com/q/3224878/1068170
[4]: http://stackoverflow.com/a/16632555/1068170

**Comment by dhermes** [comment]
Closing this out since our remaining `bash` is not used by `tox`.
